shared/css/teacher-dashboard.scss

```diff
@@ -378,3 +378,84 @@ td.studentname {
     right: 10px;
   }
 }
+
+
+.move-students-modal {
+  .modal-header {
+    padding: 15px 25px;
+  }
+
+  .section-title {
+    font-family: 'Gotham 5r', sans-serif;
+    margin-bottom: 7px;
+  }
+
+  .selection-text {
+    font-family: 'Gotham 4r', sans-serif;
+  }
+
+  .move-students-container {
+    text-align: left;
+    width: inherit;
+  }
+
+  .move-students-container .destination-section-area {
```

Comment:
I have all the CSS rules nested under `.move-students-modal`, and Hound yells at me when I add in another nesting layer, so I just removed the `.move-students-container` from these :)